frontend/src/app/_pipes/line-break-formatter.pipe.ts

```diff
@@ -0,0 +1,20 @@
+import { Pipe, PipeTransform } from '@angular/core';
+import { DomSanitizer } from '@angular/platform-browser';
+
+@Pipe({
+  name: 'LineBreakFormatter'
+})
+export class LineBreakFormatterPipe implements PipeTransform {
+  constructor(private sanitizer: DomSanitizer) {
+  }
+
+  transform(value: string): any {
```

Comment:
Is there are reason that this isn't just using "ngBindHtml" to bypass the HTML sanitization? https://docs.angularjs.org/api/ng/directive/ngBindHtml

Reply:
@lauraGgit ngBindHtml is only available in Angular 1.